source/features/more-dropdown.tsx

```diff
@@ -30,34 +30,38 @@ async function init(): Promise<void> {
 		createDropdown();
 	}
 
+	const insightsTab = select<HTMLAnchorElement>('[data-selected-links~="pulse"]')!;
+	const securityTab = select<HTMLAnchorElement>('[data-selected-links~="security"]')!;
+
 	select('.reponav-dropdown .dropdown-menu')!.append(
-		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item">
 			{icons.darkCompare()} Compare
 		</a>,
 
 		isEnterprise() ? '' :
-			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item rgh-dependency-graph" data-skip-pjax>
+			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item">
 				{icons.dependency()} Dependencies
 			</a>,
 
-		<a href={`/${repoUrl}/pulse`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
-			{icons.graph()} Insights
-		</a>,
-
-		<a href={`/${repoUrl}/commits`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/commits`} className="rgh-reponav-more dropdown-item">
 			{icons.history()} Commits
 		</a>,
 
-		<a href={`/${repoUrl}/branches`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/branches`} className="rgh-reponav-more dropdown-item">
 			{icons.branch()} Branches
 		</a>,
+
+		<a href={insightsTab.href} className="rgh-reponav-more dropdown-item">
+			{insightsTab.firstElementChild} Insights
+		</a>,
+
+		<a href={securityTab.href} className="rgh-reponav-more dropdown-item">
+			{securityTab.firstElementChild} Security
+		</a>,
 	);
 
-	// Remove native Insights tab
-	const insightsTab = select('[data-selected-links~="pulse"]');
-	if (insightsTab) {
-		insightsTab.remove();
-	}
+	insightsTab.remove();
```

Comment:
Won't this throw if the insights/security tab doesn't exist?

Reply:
I didn't think that'd ever happen. On GitHub.com it's always there I think?